examples/dynamic-enums-example.yaml

```diff
@@ -0,0 +1,138 @@
+id: https://w3id.org/linkml/examples/enums
+title: Dynamic Enums Example
+name: dynamicenums-example
+description: This demonstrates the use of dynamic enums
+license: https://creativecommons.org/publicdomain/zero/1.0/
+
+prefixes:
+  linkml: https://w3id.org/linkml/
+  ex: https://w3id.org/linkml/examples/enums/
+  sh: https://w3id.org/shacl/
+  bioregistry: https://bioregistry.io/registry/
+  MONDO: http://purl.obolibrary.org/obo/MONDO_
+  NCIT: http://purl.obolibrary.org/obo/NCIT_
+  loinc: http://loinc.org/
+
+default_prefix: ex
+default_range: string
+
+default_curi_maps:
+    - semweb_context
+
+emit_prefixes:
+  - linkml
+  - rdf
+  - rdfs
+  - xsd
+  - owl
+
+imports:
+  - linkml:types
+
+  
+#==================================
+# Classes                         #
+#==================================
+
+classes:
+  HumanSample:
+    slots:
+      - name
+      - disease
+
+
+#==================================
+# Slots                           #
+#==================================
+
+slots:
+  name:
+    range: string
+  disease:
+    range: HumanDisease
+  vital_status:
+    enum_range:
+      permissible_values:
+        LIVING:
+        DEAD:
+        UNDEAD:
+
+#==================================
+# Enums
+#==================================
+
+enums:
+  Disease:
+    reachable_from:
+      source_ontology: bioregistry:mondo
```

Comment:
You are using br for denoting sources now? Why not obo:Mondo.owl? just out of curiosity

Reply:
yeh, it may be better to do the official ontology IRI. We will need a good standard way for doing things like wikidata too